### PR TITLE
Nullifying ActiveRecord connection is no longer necessary

### LINKED ISSUE #11357: Fix master tests for runner_spec
Tests in master are failing with this trace:

```ruby
  4) CartoDB::Importer2::Runner#tracker returns the block passed at initialization
     Failure/Error: @downloader = CartoDB::Importer2::Downloader.new(@user.id, @filepath)
     ActiveRecord::RecordNotFound:
       Couldn't find Carto::User with id=40f6fccc-5c43-4497-aeca-d1b400f0d83a
     # ./services/importer/lib/importer/downloader.rb:79:in `initialize'
     # ./services/importer/spec/unit/runner_spec.rb:28:in `new'
     # ./services/importer/spec/unit/runner_spec.rb:28:in `block (2 levels) in <top (required)>'
```

### PR DESCRIPTION
**Pull request by juanignaciosl**
This closes #11357.

### CODE REVIEW

**Comment by juanignaciosl** [comment]
The last passing test probably involved some luck. The failing test is one that fails randomly (meaning that we don't know why yet). Nevertheless this should be possitive.

This won't need acceptance or deploy, as this is only tests.

File: spec/requests/api/imports_spec.rb [hunk 1]
```diff
@@ -145,18 +145,20 @@
   end
 
   it 'raises an error if the user attempts to import tables when being over quota' do
-    @user.update table_quota: 5
+    @user.update table_quota: 1
 
-    # This file contains 10 data sources
+    # This file contains 6 data sources
```
Comment (javitonino) [inline spec/requests/api/imports_spec.rb]: Lying comment...

**Comment by javitonino** [comment]
Let's wait for 🍏 in https://ci.cartodb.net/job/CartoDB-master-testing/628/console and then deploy.

**Comment by javitonino** [comment]
✅ merging!

**Pull request merged**